### DISCUSSION TITLE
Casting to other equipment on the local network

### DISCUSSION THREAD

**Discussion by ghost** [ampache/ampache#3956] [Ideas]
Are there any plant to include casting to other speakers on the network like Sonos equipment? This would be a truly valuable add-on to Ampache.

**Comment by mitchray** [comment]
There is this which may work https://github.com/hyperactivated/ampache-sonos, I don't have any equipment to test

**Reply by mitchray** [reply]
More potential options (bonob) here: https://github.com/ampache/ampache/issues/3747

**Comment by SuperFola** [comment]
As @mitchray commented, Bonob works well, once been using it with my Sonos speaker. Quite easy to configure as well

**Comment by ghost** [comment]
Thanks for your feedback but I would like to move away from the Sonos app and directly use the Sonos speaker as player from within Ampache. For example offerd by DS Audio (Synology) and by Plex (app and web ).
![image](https://github.com/ampache/ampache/assets/7385748/a0666fc1-f058-4bab-a7bf-72d6fe0cbb6b)
